kernel/Kconfig

```diff
@@ -330,6 +330,9 @@ config SYSTEM_WORKQUEUE_PRIORITY
 	default -1
 	default  0 if !COOP_ENABLED
```

Comment:
This default is outside the `range` and will get clamped to -1.

Reply:
Oh, wait, there's an `if BT` on the `range`. Maybe this can't apply in that case.

Reply:
I imagine that BT implies COOP_ENABLED because there was an assertion on this value being negative in BT.